docs/administration/back_office/search_suggestion.md

```diff
@@ -0,0 +1,53 @@
+---
+description: Customize search suggestion configuration and sources.
+---
+
+# Customize search suggestion
+
+In the Back Office, when typing some text in the search bar, some suggestions about what you could be looking for are made directly under this bar. See [user documentation about content search]([[= user_doc =]]/search/search_for_content) for feature usage.
+
+## Configuration
+
+By default, the suggestions start to be made when the typed text is at least 3 characters long, and 5 suggestions are made.
```

Comment:
```suggestion
By default, the suggestions start to be made when the typed text is more than 3 characters long, and 5 suggestions are made.
```
Because of https://github.com/ibexa/admin-ui/pull/901/files#diff-fcfdc22ac6b21e48a6e9bab53b268b8876479aa3bae9b37de117feb987d12d3cR77 `<=`, `…search.suggestion.min_query_length` parameter name is counter-intuitive.

Reply:
To have a parameter coherent with its name, suggestion should start at `min_query_length`, not `min_query_length+1`
See ibexa/admin-ui#997 for fix proposal